script.js

```diff
@@ -0,0 +1,14 @@
+function getTriangleArea(a, h) {
+
+	if (a > 0 && h > 0) {
+	  return a*h/2;
+	} else {
+		return "Nieprawidłowe dane";
```

Comment:
`else` is unnecessary here, because return in if stopped your function when a and b will be greater than `0`

Reply:
napisz proszę po Polsku bo nie jestem pewien czy dobrze rozumiem. (else usunięte)

Reply:
ok, else usunąłeś ale samo return powinno zostać, bo funkcja powinna coś zwracać